### DISCUSSION TITLE
Manipulating repeatable components

### DISCUSSION THREAD

**Discussion by dodo007** [strapi/strapi#6231] [Legacy Topics]
Does anybody know how can I create repeatable component instances when using MongoDB as a database solution?

I need a way to create new instances of components and update existing instances of components from the code. 
I did not find anything in the docs.

***System config***
Operating system: Ubuntu 16.04
Database: MongoDB 2.6.10
Strapi version: Strapi 3.0.0-beta.20.3
Node v12.16.3
NPM 6.14.4

Thank you.

**Comment by lauriejim** [comment]
Hello! Can you give an example with your Content Type name and Components name.
I'm not sure to understand what you are trying to do.
